doc/source/user_guide/tutorials/mathematics/basic_maths.rst

```diff
@@ -0,0 +1,814 @@
+.. _ref_basic_math:
+
+===========
+Basic maths
+===========
+
+.. |Field| replace:: :class:`Field<ansys.dpf.core.field.Field>`
+.. |Fields| replace:: :class:`Field<ansys.dpf.core.field.Field>`
+.. |FieldsContainer| replace:: :class:`FieldsContainer<ansys.dpf.core.field.Field>`
+.. |FieldsContainers| replace:: :class:`FieldsContainer<ansys.dpf.core.field.Field>`
```

Comment:
Hm yes we can make it in a separate file and include it in the beginning of each tutorial. But then it would put references we don't use in some tutorials. Idk if that ok ?